qtglean/src/ffi/string.rs

```diff
@@ -19,14 +19,14 @@ pub extern "C" fn glean_string_test_get_value(
     id: u32,
     ping_name: FfiStr
 ) -> *mut c_char {
-    let returnString = with_metric!(
+    let return_string = with_metric!(
```

Comment:
This resolves a style-related build warning that I missed when I created this metric.